flytekit/clis/flyte_cli/main.py

```diff
@@ -30,11 +32,30 @@
 
 _tt = _six.text_type
 
+# Similar to how kubectl has a config file in the users home directory, this Flyte CLI will also look for one.
+# The format of this config file is the same as a workflow's config file, except that the relevant fields are different.
+# Please see the example.config file
+_default_config_file_path = ".flyte/config"
```

Comment:
I think this is already set as ‘~/.Flyte’ in the configuration.internal. Perhaps use that, but I’d also support changing if to ‘~.flyte/config’

Reply:
Looks like it's just flytekit.config https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/configuration/internal.py#L13 - do we want to change the default as part of this change? i'm worried about unrolling out too much one in go (and breaking stuff in the process)

Reply:
Yeah, I think that's probably best. Looking at the way the code is written below, I think it's good actually. It will fall back to `flytekit.config` when `~/.flyte/config` doesn't exist so there isn't any breaking change (and if that isn't found, which i think 99.99% likely it won't be found, it will go to env vars).